CHANGES.md

```diff
@@ -31,7 +31,7 @@
 
 ## I/Os
 
-* Support for X source added (Java/Python) ([#X](https://github.com/apache/beam/issues/X)).
+* Debezium IO upgraded to 3.1.1 requires Java 17 (Java) ([#34747](https://github.com/apache/beam/issues/34747)).
```

Comment:
Please move it under 2.66.0 Section. Here it is template.

Reply:
Thank you, done!